src/content/docs/en/guides/integrations-guide/vue.mdx

```diff
@@ -211,3 +211,21 @@ export default defineConfig({
   integrations: [vue({ devtools: true })],
 });
 ```
+
+For more customizations you can also pass all the options
+that the [Vue DevTools Vite Plugin](https://devtools-next.vuejs.org/guide/vite-plugin#options) supports
+(except appendTo). 
```

Comment:
```suggestion
#### Customizing Vue DevTools

<p><Since pkg="@astrojs/vue" v="4.3.0" /></p>

For more customization, you can instead pass options that the [Vue DevTools Vite Plugin](https://devtools-next.vuejs.org/guide/vite-plugin#options) supports. (Note:  `appendTo` is not supported.)

For example, you can set `launchEditor`  to your preferred editor if you are not using Visual Studio Code:
```
Not sure a section header was REALLY necessary here, but it works better if we want to add a Since with a different minor version, so I think this works.